### PR TITLE
Make CLI help descriptions and usage a bit more consistent

### PR DESCRIPTION
**Pull request by nalind**
Start with capital letters; a description ends with a period; usage doesn't.

### CODE REVIEW

File: cmd/buildah/umount.go [hunk 1]
```diff
@@ -20,8 +20,8 @@ var (
 	umountCommand = cli.Command{
 		Name:                   "umount",
 		Aliases:                []string{"unmount"},
-		Usage:                  "Unmounts the root file system on the specified working containers",
-		Description:            "Unmounts the root file system on the specified working containers",
+		Usage:                  "Unmount the root file system on specified working containers",
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/umount.go]: I think "the' should remain.

File: cmd/buildah/umount.go [hunk 2]
```diff
@@ -20,8 +20,8 @@ var (
...
+		Description:            "Unmounts the root file system on the specified working containers.",
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/umount.go]: Singular Unmount here too?

Comment (TomSweeneyRedHat) [reply cmd/buildah/umount.go]: Oops, never mind this one, I see the pattern here.

**Comment by TomSweeneyRedHat** [comment]
One small nit and Red Hat CI isn't happy, probably unrelated.  Otherwise LGTM.

**Comment by rhatdan** [comment]
LGTM  Just fix @TomSweeneyRedHat comment.

File: cmd/buildah/umount.go (hunk 1 above)
Comment (nalind) [reply cmd/buildah/umount.go]: Changed.  That "on" should have been an "of", though, so changed that, too.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**